literature_search.py

```diff
@@ -158,26 +174,33 @@ def clean_str(abs_string,stop):
 indx = []
 
 for k, top_val in enumerate(prediction_vec):
-    pred_val = np.max(top_val)
-    if pred_val > 0*np.sort(top_val)[-2]:
-        indx.append(k)
-        topics.append(le.inverse_transform([np.argmax(top_val)])[0])
-        title_temp.append(papers_df['title'][k])
-    else:
-        indx.append(k)
-        topics.append('unknown')
-        title_temp.append(papers_df['title'][k])
+    try:
+        papers_df.loc[k,:]
+        pred_val = np.max(top_val)
+        if pred_val > 0*np.sort(top_val)[-2]:
+            indx.append(k)
+            topics.append(le.inverse_transform([np.argmax(top_val)])[0])
+            title_temp.append(papers_df['title'][k])
+        else:
+            indx.append(k)
+            topics.append('unknown')
+            title_temp.append(papers_df['title'][k])
+    except:
+        print('Skipping prediction of paper #: ' + str(k))
 papers_df = pd.DataFrame(data = {'title': title_temp,
                                   'topic': topics})
-pd.set_option('display.max_colwidth', -1) #don't cut cell short in dataframe
-papers_df[['title','topic']].sample(5)
-pd.reset_option('display.max_colwidth')
 
 #==============================================================================
 #========================= Save Titles and Topics =============================
 
 #add info for github markdown format
 papers_df['title']   = [title if title[1] is not '[' else title[1:-1] for title in papers_df['title']]
+title_tmp = []
+for k, title in enumerate(papers_df['title']):
+    if title[1] is not '[':
+        title_tmp.append(title)
+    else:
+        title_tmp.append(title[1:-1])
```

Comment:
what is the plan for `title_tmp` ?

Reply:
Readability. List comp is cool, but really annoying to debug, so was usign it to debug. I'll remove it